integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java

```diff
@@ -137,36 +151,22 @@ public void setMetalakeProps(int index, String key, String value) {
     valueInput.sendKeys(value);
   }
 
-  public boolean checkIsErrorName() {
-    String xpath = "//div[@data-refer='metalake-name-field']";
-    WebElement nameField = findElementByXPath(xpath);
-    List<WebElement> errorText =
-        nameField.findElements(By.xpath("//div[contains(@class, 'Mui-error')]"));
+  public boolean checkIsErrorName() throws InterruptedException {
+    try {
+      String xpath = "//div[@data-refer='metalake-name-field']";
+      WebElement nameField = driver.findElement(By.xpath(xpath));
```

Comment:
Can we define a link element variable in class member?

Reply:
Done